scripts/release/goa_prepare/sets_as_physical_entities.pl

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+
+use lib "/usr/local/gkb/modules";
+use GKB::CommonUtils;
+
+use autodie;
+use Getopt::Long;
+use List::MoreUtils qw/any notall/;
+
```

Comment:
Descriptor for this function would be useful, if this will be used moving forward

Reply:
Sure, I will add a comment beside the use statement